cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -744,6 +746,49 @@ pub struct ManifestServiceDescriptor {
     pub command: String,
     /// Service-specific environment variables
     pub vars: Option<ManifestVariables>,
+    /// Whether the service spawns a background process (daemon)
+    // TODO: This option _requires_ the shutdown command, so we'll need to add
+    //       that explanation to the manifest.toml docs and service mgmt guide
+    pub is_daemon: Option<bool>,
+    /// How to shut down the service
+    pub shutdown: Option<ManifestServiceShutdown>,
+}
+
+impl ManifestServices {
+    pub fn validate(&self) -> Result<(), ServiceError> {
+        let mut bad_services = vec![];
+        for (name, desc) in self.0.iter() {
+            let daemonizes = desc.is_daemon.is_some_and(|_self| _self);
+            let has_shutdown_cmd = desc.shutdown.is_some();
+            if daemonizes && !has_shutdown_cmd {
+                bad_services.push(name.clone());
+            }
+        }
+        let list = bad_services
+            .into_iter()
+            .map(|name| format!("- {name}"))
+            .join("\n");
+        if list.is_empty() {
+            Ok(())
+        } else {
+            let msg = formatdoc! {"
+                            Services that spawn daemon processes must supply a shutdown command.
+
+                            The following services did not specify a shutdown command:
+                            {list}                            
+                        "};
```

Comment:
nit/suggestion: reduce indent
```suggestion

                Services that spawn daemon processes must supply a shutdown command.

                The following services did not specify a shutdown command:
                {list}
            "};
```